Description: Update configure.in file to current format for autoconf version 2.x.
 * Modernize the configure.in file to current autoconf style.
 * No need to rename to configure.ac since configure.in still works fine.
Author: Bob Proulx

--- time-1.7.orig/configure.in
+++ time-1.7/configure.in
@@ -1,11 +1,7 @@
 dnl Process this file with autoconf to produce a configure script.
-AC_INIT(time.c)
-VERSION=1.7
-AC_SUBST(VERSION)
-PACKAGE=time
-AC_SUBST(PACKAGE)
 
-AC_ARG_PROGRAM
+AC_INIT([time],[1.7])
+AM_INIT_AUTOMAKE([-Wall])
 
 dnl Checks for programs.
 AC_PROG_CC
@@ -39,4 +35,5 @@
 AC_MSG_RESULT($have_gp)
 test $have_gp = yes && AC_DEFINE(HAVE_GETPAGESIZE)
 
-AC_OUTPUT(Makefile)
+AC_CONFIG_FILES([Makefile])
+AC_OUTPUT
